Header header
float64 StartAngle
float64 EndAngle
velodyne/ScanMsg[] Scans
